src/modules/mavlink/mavlink_receiver.cpp

```diff
@@ -2519,9 +2533,12 @@ MavlinkReceiver::receive_thread(void *arg)
 		}
 	}
 
-#ifdef __PX4_POSIX
+#if defined(__PX4_POSIX)
 	/* 1500 is the Wifi MTU, so we make sure to fit a full packet */
 	uint8_t buf[1600 * 5];
```

Comment:
Can you handle the buffer size consistent for each platform with setting BUFF_SIZE? Please also add a comment explaining the size.

Reply:
Actually this change was made quite a while ago, it was one of the first changes in order to make the Ethernet to work and the size of the buffer wasn't analyzed properly.
My thought is maybe it should be Mavlink payload max size + header size + checksum size,  what do you think?

Reply:
@dagar